package.json

```diff
@@ -29,10 +29,10 @@
     "start": "open http://uber.github.io/deck.gl/#/documentation/getting-started/installation?section=running-the-examples",
     "test": "scripts/test.sh",
     "test-fast": "scripts/test.sh fast",
-    "test-browser": "webpack-dev-server --config test/webpack.config.js --env.test_browser --progress --hot --open",
-    "test-render": "webpack-dev-server --config test/webpack.config.js --env.render --progress --hot --open",
+    "test-browser": "node test/start browser",
+    "test-render": "node test/start render",
```

Comment:
need keep the browser open when test is done

Reply:
`browser-headless` is the headless mode. `browser` keeps the window open.